models/marts/int_custoemrs_no_orders.sql

```diff
@@ -0,0 +1,22 @@
+{%- set status = ['return_pending','returned','complete'] -%}
+with customers as (
+   select * from {{ ref('stg_customers') }}
+),
+orders as (
+   select * from {{ ref('stg_orders') }}
+),
+ 
+final as (
+   select c.customer_id,
+     {% for status in status -%}
+    sum(case when status = '{{ status }}' then 1 else 0 end) 
+            as {{ status }}_count    
+       {%- if not loop.last -%}
+         ,
+       {% endif -%}
+        {%- endfor %}
+    from customers as c join orders as o on c.customer_id=o.customer_id

```

Comment:
We are strong believers in not alias-ing on initials (because it takes away from the readability of SQL). I would keep this as orders and customers. 

[Feel free to take a look at our style guide for more best practices](https://github.com/fishtown-analytics/corp/blob/master/dbt_coding_conventions.md#sql-style-guide)